chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -192,6 +192,21 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
             endChat(props, chatSDK, setAdapter, setWebChatStyles, dispatch, adapter, false, false, false);
         });
 
+        // Close popout window
+        BroadcastService.getMessageByEventName("ClosePopoutWindow").subscribe(() => {
```

Comment:
Where is this event triggered?

Reply:
This is supposed to be triggered in the script layer after the popout window is closed